src/test/java/net/sf/jabref/gui/autocompleter/AutoCompleterTest.java

```diff
@@ -0,0 +1,56 @@
+package net.sf.jabref.gui.autocompleter;
+
+import net.sf.jabref.JabRef;
+import net.sf.jabref.gui.entryeditor.EntryEditor;
+import net.sf.jabref.gui.fieldeditors.FieldEditor;
+import net.sf.jabref.gui.fieldeditors.TextArea;
+import net.sf.jabref.logic.autocompleter.AutoCompleter;
+import net.sf.jabref.model.entry.BibtexEntry;
+import net.sf.jabref.testutils.TestUtils;
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.awt.event.ActionEvent;
+
+public class AutoCompleterTest {
+
+    @Test
+    @Ignore(value = "works on windows and ubuntu, but not on travis")
```

Comment:
Hm, I prefer the ignore here because this test requires work to be fixed. This is not an issue with an external service as we had in the fetcher cases.